eslint.config.mjs

```diff
@@ -0,0 +1,28 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
I think since node 20 we can also use `import.meta.dirname` directly (at least it works in vite and vitest configs)
https://nodejs.org/api/esm.html#importmetadirname

Reply:
indeed, thanks a lot :)